analyses/cms-open-data-ttbar/requirements.txt

```diff
@@ -0,0 +1,146 @@
+aiohttp==3.8.1
+aiosignal==1.2.0
+aiostream==0.4.4
+argon2-cffi==21.3.0
+argon2-cffi-bindings==21.2.0
+asttokens==2.0.5
+async-timeout==4.0.2
+attrs==21.4.0
+awkward==1.8.0
+awkward0==0.15.5
+backcall==0.2.0
+backoff==2.1.2
+beautifulsoup4==4.11.1
+bleach==5.0.1
+boost-histogram==1.3.1
+cabinetry==0.4.1
+cachetools==5.2.0
+certifi==2022.6.15
+cffi==1.15.1
+charset-normalizer==2.1.0
+click==8.1.3
+cloudpickle==2.1.0
+coffea==0.7.16
+commonmark==0.9.1
+configparser==5.2.0
+confuse==1.3.0
+correctionlib==2.1.0
+cycler==0.11.0
+dask==2022.6.1
+debugpy==1.6.1
+decorator==5.1.1
+defusedxml==0.7.1
+distributed==2022.6.1
+entrypoints==0.4
+executing==0.8.3
+fastjsonschema==2.15.3
+fonttools==4.34.3
+frozenlist==1.3.0
+fsspec==2022.5.0
+func-adl==3.0
+func-adl-servicex==2.0
+google-auth==2.9.0
+HeapDict==1.0.1
+hist==2.6.1
+histoprint==2.4.0
+idna==2.10
+iminuit==2.12.1
+importlib-resources==5.8.0
+ipykernel==6.15.0
+ipython==8.4.0
+ipython-genutils==0.2.0
+ipywidgets==7.7.1
+jedi==0.18.1
+Jinja2==3.1.2
+jsonpatch==1.32
+jsonpointer==2.3
+jsonschema==4.6.1
+jupyter==1.0.0
+jupyter-client==7.3.4
+jupyter-console==6.4.4
+jupyter-core==4.10.0
+jupyterlab-pygments==0.2.2
+jupyterlab-widgets==1.1.1
+kiwisolver==1.4.3
+lark-parser==0.12.0
+llvmlite==0.38.1
+locket==1.0.0
+lz4==4.0.1
+make-it-sync==1.0.0
+MarkupSafe==2.1.1
+matplotlib==3.5.2
+matplotlib-inline==0.1.3
+minio==5.0.10
+mistune==0.8.4
+mplhep==0.3.26
+mplhep-data==0.0.3
+msgpack==1.0.4
+multidict==6.0.2
+nbclient==0.6.6
+nbconvert==6.5.0
+nbformat==5.4.0
+nest-asyncio==1.5.5
+notebook==6.4.12
+numba==0.55.2
+numpy==1.21.6
+packaging==21.3
+pandas==1.4.3
+pandocfilters==1.5.0
+parso==0.8.3
+partd==1.2.0
+pexpect==4.8.0
+pickleshare==0.7.5
+Pillow==9.2.0
+pkg_resources==0.0.0
```

Comment:
I think this shouldn't be in here, at least it causes issues when trying to `python -m pip install -r requirements.txt`:
```
ERROR: No matching distribution found for pkg_resources==0.0.0
```

Reply:
that's interesting, I don't get this error. Using Python 3.8.10

Reply:
The behavior might be OS-specific: https://stackoverflow.com/questions/39577984/what-is-pkg-resources-0-0-0-in-output-of-pip-freeze-command. As far as I can tell from this it should be safe to remove it.

Reply:
interesting. I removed it